setup.py

```diff
@@ -136,6 +135,9 @@ def run(self):
                         'sortedcontainers==2.1.0',
                         ### Tests fail without version pin (GHA run: https://github.com/udosson/indy-plenum/actions/runs/1078741118)
                         'ujson==1.33',
+                        'indy_vdr==0.4.2', 
+                        'aries-askar==0.4.3', 
+                        'indy-credx==1.1.1'
```

Comment:
This makes the indy-plenum deployment package dependent on client side components that are only used for tests.